src/core/client_config/resolvers/dns_resolver.c

```diff
@@ -42,8 +42,14 @@
 #include "src/core/client_config/lb_policy_registry.h"
 #include "src/core/iomgr/resolve_address.h"
 #include "src/core/iomgr/timer.h"
+#include "src/core/support/backoff.h"
 #include "src/core/support/string.h"
 
+#define BACKOFF_MULTIPLIER 1.6
```

Comment:
I'd argue that we should remove the prefixes in subchannel.c (no need to prefix private symbols)